py-polars/tests/unit/io/test_iceberg.py

```diff
@@ -163,3 +163,12 @@ def test_parse_lteq(self) -> None:
 
         expr = _to_ast("(pa.compute.field('ts') <= '2023-08-08')")
         assert _convert_predicate(expr) == LessThanOrEqual("ts", "2023-08-08")
+
+    def test_compare_boolean(self) -> None:
```

Comment:
Can you also add a test to validate you get the correct result?

Reply:
I would like to, yes. As I mentioned in the last paragraph of the PR description I’m not really sure how to go about that. Is there some other way that doesn’t involve recreating the iceberg table fixture with a new Boolean column?

Reply:
I also don't really know about iceberg. Ok, let's merge for now.